### ISSUE TITLE
Tag new release

### ISSUE THREAD

**Issue by AMDmi3** [nevat/griels-quest#3]
Could you please tag latest trunk as a new release? It will make packaging easier, as master branch contains crucial fixes after the latest tag.

**Comment by AMDmi3** [comment]
Cast @carstene1ns

**Comment by carstene1ns** [comment]
Yeah, at least the packaging is much easier now. Will tag a minor release (1.0.1) later.

**Comment by AMDmi3** [comment]
Ping?

**Comment by AMDmi3** [comment]
Ping again?

**Comment by AMDmi3** [comment]
Ping again again :(

**Comment by AMDmi3** [comment]
Another ping
